### ISSUE TITLE
New Topnav

### ISSUE THREAD

**Issue by dylan-k** [real-pants/Real-Pants#77]
for version 1.2 we want to be sure that the topnav appears everywhere on the site. in v1.1 there's an issue where [the menu fails to appear on category pages](https://wordpress.org/support/topic/wp-nav-menu-dissapears-in-category-pages-1?replies=21). I'm not certain of the cause but it may go as far back as #9.

Early upon discovering this issue, Adam and I thought we might want to quit using the UberMenu plugin for wordpress, for simplicity's sake. We thought perhaps [YAMM3](http://geedmo.github.io/yamm3/) would be a better fit. As it turned out, it would be more work to implement that into wordpress than it's worth.

**Comment by dylan-k** [comment]
We talked about maybe adding this instead of the ubermanu plugin that's in place now: http://geedmo.github.io/yamm3/

It would need to be configured via wordpress admin.,

**Comment by dylan-k** [comment]
see also: http://www.hongkiat.com/blog/bootstrap-megamenu-yamm3/

**Comment by dylan-k** [comment]
Tonight I've started building version 1.2 of the theme for realpants.com.

For this build, I'm starting from a fresh copy of the framework that I started from originally. This is because that framework has been substantially rebuilt since then and among the revisions is a new way of handling the topnav, which is useful for this issue.

As of tonight, logged in users will get to see 1.2 in its current state.
Standard users will continue to see 1.1, though it may not behave perfectly.
I'll want to merge the good stuff over from 1.1 into 1.2, but already I can see that 1.2 lacks this issue with the topnav.

**Comment by realpants** [comment]
*This may or may not be applicable to the previous issue with the Topnav.*

While rebuilding plugins and widgets on the site, the following occurred:
* I activated Ubermenu and did not need to rebuild anything, everything appeared as it was before the meltdown
* I activated the Ubermenu Sticky extension, all good. The menu was fixed to the top of the window.

In fact, at this point, functionality seemed perfect; I tested it and the menu appeared on all category pages, and stayed sticky. 

* Then I wanted to replace the menu in the footer. I put the Custom Menu widget in the footer, set it to "Beats List," and checked the site.
* I noticed, first of all, that none of the items appeared in the menu.
* Further, and more relevantly, I noticed that the Ubermenu was broken: it was no longer sticky.
* Additionally, the WP Toolbar was gone. The space for it was still there (ie. the Ubermenu was set 30px [or whatever the height of the Toolbar is] down from the top of the window)

My conclusion is that there is a conflict with the Custom Menu widget and Ubermenu. I tested other widgets in the footer and the issue did not occur.
